### ISSUE TITLE
"Schema validation error" when trying to build the template

### ISSUE THREAD

**Issue by SusiKette** [madskristensen/TextmateExtensionTemplate#1]
I'm getting the following error when I try to build the template:

`Schema validation error for 'C:\Users\Scott\source\repos\testing\testing\obj\Debug\extension.vsixmanifest'. The http://schemas.microsoft.com/developer/vsx-schema/2011:Assets element is not declared.`

How do I fix this?

**Comment by ComotoJC** [comment]
yeah I got past this by editing the vsimanifest, and fixing the  install targets to be 14.0, 18.0 and same for pre-requisites for core editor.
